Dockerfile

```diff
@@ -1,10 +1,10 @@
 # NOTE: Must be run in the context of the repo's root directory
 
 ## (1) Download a suitable version of Go
-# FIX: We use 1.17 due to unsupported modules
+# FIX: We use 1.18 due to unsupported modules

```

Comment:
guess you can remove this comment now :)

Reply:
@A-town21 Does it build correctly? We can remove it, if there are no more issues. Thanks.

Reply:
Yes, removed. Thanks.